### PR TITLE
feat(button-toggle): enhance button toggle visuals - FE-4166

### PR DESCRIPTION
**Pull request by mkrds**
### Proposed behaviour
This PR enhances ButtonToggle visuals

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

File: src/components/button-toggle/button-toggle.style.js [hunk 1]
```diff
@@ -14,43 +44,50 @@ const StyledButtonToggleContentWrapper = styled.div`
 const StyledButtonToggleLabel = styled.label`
   display: inline-block;
   position: relative;
-  height: 38px;
-  padding: 0 24px;
+  box-sizing: border-box;
+  height: ${({ size }) => `${heightConfig[size]}px`};
+  padding: ${({ size }) => `0 ${paddingConfig[size]}px`};
+  font-size: ${({ size }) => `${fontSizeConfig[size]}px`};
```
Comment (sjohnson-sage) [inline src/components/button-toggle/button-toggle.style.js:50]:
```javascript
${({ size }) => css`
  height: ${heightConfig[size]}px;
  padding: 0 ${paddingConfig[size]}px;
  font-size: ${fontSizeConfig[size]}px;
`}
```

would be better IMO

**Review by sjohnson-sage — COMMENTED** [review]
Just one minor comment, and I think the commit message should reference the new `size` prop so it's in the release notes

**Review by dawidzarzycki — APPROVED** [review]
Nothing more than Sam posted

**Review by sjohnson-sage — APPROVED** [review]

**Review by sjohnson-sage — APPROVED** [review]

**Comment by harpalsingh** [comment]
Just to add a note on some Button Toggle Group behaviour, currently when using keyboard and arrow keys, once you move left/right it automatically makes the selection "Active" and therefore you never see the Inactive + focus state. I was happy to allow this behaviour to continue for now, but feel similar to tabs, it should be an option you can toggle on/off, will speak with Accessibility team about the best approach.

**Review by DlgSHi — APPROVED** [review]

**Pull request merged**